### DISCUSSION TITLE
How to specify which RuleSet to use on a form?

### DISCUSSION THREAD

**Discussion by mgwilliam** [Blazored/FluentValidation#98] [Q&A] [answered]
I have a model and a validator that has 2 RuleSets. 
Each RuleSet applies different validation to the same fields in the model.
How can I specify which RuleSet to use in the form? Currently it seems to be running all of the rules and producing validation messages from both RuleSets.

**Comment by chrissainty** [comment, marked as answer]
This isn't something you can do right now, I'm working on an update to bring this feature in. It is currently tracked by #58

**Comment by mgwilliam** [comment]
Thanks @chrissainty 
I've worked around it for now using a single base validator and then deriving 2 empty subclasses from it (effectively 1 per rule set)
